code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -585,7 +585,7 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 /obj/item/restraints/handcuffs/energy
 	name = "hard-light energy field"
 	desc = "A hard-light field restraining the hands."
-	icon_state = "cuff" // Needs sprite
+	icon_state = "handcuffAlien" //likely sprite change, enabling 
 	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
 	breakouttime = 450
```

Comment:
this will make cult handcuffs also use the new sprite

Reply:
well isnt it a spell?

Reply:
yes, but it spawns handcuffs

Reply:
ill hide it then

also i converted the handcuffs page over to seconds